homeassistant/components/generic/strings.json

```diff
@@ -55,6 +55,7 @@
           "authentication": "[%key:component::generic::config::step::user::data::authentication%]",
           "limit_refetch_to_url_change": "[%key:component::generic::config::step::user::data::limit_refetch_to_url_change%]",
           "password": "[%key:common::config_flow::data::password%]",
+          "use_wallclock_as_timestamps": "Rewrite timestamps",
```

Comment:
Is it possible to provide slightly more explanation of what this is?  How does the user decide what to set it to and can we hint at this in the interface somehow?